src/lib/data/creatables/slayer.ts

```diff
@@ -130,7 +132,16 @@ export const slayerCreatables: Createable[] = [
 		}),
 		outputItems: new Bank({ 'Abyssal tentacle': 1 }),
 		GPCost: 0,
-		maxCanOwn: 1
+		maxCanOwn: 1,
```

Comment:
```suggestion
		maxCanOwn: 1,
		
```
Can be removed with this PR right?

Reply:
It technically could be removed, however then you get into a situation where there's multiple Abyssal tentacles, all sharing the same charge pool, which I don't think is the best idea considering they degrade into just the tentacle.

Reply:
Yea I see what you mean. Then it looks good. 🥰🐳